prow/config.yaml

```diff
@@ -4327,6 +4372,54 @@ presubmits:
       - emptyDir: {}
         name: auto-generated-docker-graph-volume-mount
     trigger: (?m)^/test pull-security-kubernetes-kubemark-e2e-gce-scale,?(\s+|$)
+  - agent: kubernetes
```

Comment:
nit: can you pull the `name: pull-security-kubernetes-local-e2e` up here?

Reply:
i believe the file is sorted